sdk/keyvault/azure-security-keyvault-secrets/tspconfig.yaml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  "service-dir":
+    default: "sdk/keyvault"
+
+emit:
+  - "@azure-tools/typespec-autorest"
+
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
+  "@azure-tools/typespec-cpp":
+    flavor: azure
```

Comment:
Do we actually need each one of these config options?

I understand `package-dir` and `namespace`, but am not sure if the above actually impact the generated code in some way.